pkg/u-boot/Dockerfile

```diff
@@ -8,7 +8,9 @@ SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
 ENV VERSION v2024.10-rc2
 ENV SOURCE_URL https://github.com/u-boot/u-boot/archive/${VERSION}.tar.gz
 ENV RASPBERRY_FIRMWARE_BLOBS_VERSION 1.20211007
+ENV RASPBERRY_FIRMWARE_BLOBS_VERSION2 1.20240306
```

Comment:
@dpoulios I think we can use the newest firmware for RPi model 4 as well, right? So you can just update the version on RASPBERRY_FIRMWARE_BLOBS_VERSION...

Reply:
Ok, I will do it this way and test on rpi4 & rpi5

Reply:
I tested on RPI4, RPI5 & OnLogic FR201 (rpi cm4) using the latest firmware files (version 1.20240306 for all binary files we download) and although rpi4 & rpi5 seemed ok,  I saw some issues in u-boot in FR201. So, I'm suggesting to keep this as it is now and revisit it again in the future.